adapters/sovrn/sovrntest/video/multi-banner-video.json

```diff
@@ -0,0 +1,185 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id1",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
```

Comment:
Consider the imp order reversed in this test case where the first imp is a video imp and the second one is the banner imp, in that case the bid type in the response will be incorrectly set to `video`.. Now also consider the imps in the same order as you have in this test, if your bidder service were to send back a bid for the video imp as well, in this case the video bid would have the bid type incorrectly set to `banner`

Reply:
I believe this has been solved by using the response `Bid.impid` value.